packages/occ-worker/src/actions.ts

```diff
@@ -58,6 +59,13 @@ function buildModel(
           };
           break;
         }
+        case JCadWorkerSupportedFormat.STL: {
+          shapeData = shapeFactory['Post::Operator']?.(
+            parameters as IOperatorArg,
+            model
+          );
+          break;
+        }
```

Comment:
Small nitpick, let's reuse the other case block:

Small nitpick

```suggestion
        case JCadWorkerSupportedFormat.BREP: 
        case JCadWorkerSupportedFormat.STL: {
          shapeData = shapeFactory['Post::Operator']?.(
            parameters as IOperatorArg,
            model
          );
          break;
        }
```